### ISSUE TITLE
[Enhancement] Mirrormaker to provide offset.commit.interval.ms

### ISSUE THREAD

**Issue by weichuliu** [strimzi/strimzi-kafka-operator#1734]
**Is your feature request related to a problem? Please describe.**
Hi, we are using Strimzi to provision mirrormakers. However seems the `offset.commit.interval.ms` is not configurable so mirrormakers are committing at an interval of 60 seconds. This is a bit too long for our use case so we want to tune the configuration.

**Describe the solution you'd like**
Add a `offsetCommitIntervalMs` (probably this name), to configure the specific value.

**Additional context**

`offset.commit.interval.ms` is a mirrormaker specific configuration. It is neither producer conf or consumer conf.
It can be found at

```
kafka/bin/kafka-mirror-maker.sh |& grep -A3 offset
--offset.commit.interval.ms <Integer:    Offset commit interval in ms.
  offset commit interval in                (default: 60000)
  millisecond>
--producer.config <String: config file>  Embedded producer config.
--rebalance.listener.args <String:       Arguments used by custom rebalance
```

**Comment by scholzj** [comment]
Have you tried to just add it to the consumer config section? Does that not
work?
https://strimzi.io/docs/latest/full.html#assembly-kafka-mirror-maker-configuration-deployment-configuration-kafka-mirror-maker

**Comment by weichuliu** [comment]
@scholzj Need to double confirm but if I remember correctly, mirrormaker disabled auto commit for its consumer. Instead it is commiting the offset by itself.

So consumer config probably won't help.

**Comment by scholzj** [comment]
Ok, get it now. Thanks for the explanation. I agree that this should be added.

**Comment by scholzj** [comment]
Thanks for raising this issue. It should be now fixed in #1760. #1768 will add the docs for this.

**Comment by weichuliu** [comment]
That is awesome! Thanks for the fix!
